gulp/utils/configHelpers.js

```diff
@@ -0,0 +1,22 @@
+const glob = require('glob');
+
+module.exports = {
+  getAvailableLanguages: (config) => {
+    const entry = config.entryGlobs.content;
+    const entryPath = config.entryPaths.content;
+    return entry.map(e => glob.sync(e))
+      .reduce((a, b) => a.concat(b), [])
```

Comment:
I think explicit code is much better.
In the first example I don't event know that glob.sync returns a path string. I have to guess or go to it's doc.